src/commandExecutor.ts

```diff
@@ -98,19 +99,39 @@ const CommandExecutor = async () => {
   }
 
   const createSignedTx = async (args: ParsedTransactionSignArguments) => {
-    validateRawTxBeforeSigning(args.rawTxFileData.cborHex)
-    const rawTxCbor = Buffer.from(args.rawTxFileData.cborHex, 'hex')
-    const rawTx = InteropLib.decodeRawTx(rawTxCbor)
+    // eslint-disable-next-line no-console,max-len
+    console.log('Warning! This call is DEPRECATED and will be REMOVED in Oct 2022. Please use witness call instead.')
+
+    let rawTx: InteropLib.RawTransaction | undefined
+    let tx: InteropLib.Transaction | undefined
+    if (args.rawTxFileData) {
+      validateRawTxBeforeSigning(args.rawTxFileData.cborHex)
+      // eslint-disable-next-line no-console,max-len
+      console.log('Warning! The --tx-body-file option is DEPRECATED and will be REMOVED in Oct 2022. Please use --tx-file instead (use --cddl-format when building transactions with cardano-cli).')
+
+      const rawTxCbor = Buffer.from(args.rawTxFileData.cborHex, 'hex')
+      rawTx = InteropLib.decodeRawTx(rawTxCbor)
+    } else {
+      validateTxBeforeSigning(args.txFileData!.cborHex)
+      const txCbor = Buffer.from(args.txFileData!.cborHex, 'hex')
+      tx = InteropLib.decodeTx(txCbor)
+    }
+
+    const txBody = (rawTx?.body ?? tx?.body)!
```

Comment:
This is rather ugly, but I lean towards doing it this way instead of creating a proper union type, since we will remove `rawTxFileData` altogether in a few months.